source/Grid/Grid.test.js

```diff
@@ -1507,4 +1508,33 @@ describe('Grid', () => {
       expect(cellRendererCalls[1].style.width).toEqual(50)
     })
   })
+
+  it('should not pull from the style cache while scrolling if there is an offset adjustment', () => {
+    let cellRendererCalls = []
+    function cellRenderer (props) {
+      cellRendererCalls.push(props)
+    }
+
+    const grid = render(getMarkup({
+      cellRenderer,
+      width: 100,
+      height: 100,
+      rowHeight: 100,
+      columnWidth: 100,
+      rowCount: DEFAULT_MAX_SCROLL_SIZE * 2 / 100, // lots of offset
```

Comment:
Ah, I see. I think the order of operations threw me a bit. (It's late here.) It's fine as is. :)